src/test/java/github/weichware10/analyse/HeatmapTest.java

```diff
@@ -15,15 +15,15 @@
      */
     @Test
     public void createHeatmapShouldWork() {
-        Heatmap hm1 = new Heatmap(new Data(ToolType.ZOOMMAPS, 1), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(new TrialData(ToolType.ZOOMMAPS, "1", "3"), new ConfigHeatmap());
         assertEquals("createHeatMap sollte .../heatmap/HEATMAP_ZOOMMAPS_1.jpg ausgeben",
                 ".../heatmap/HEATMAP_ZOOMMAPS_1.jpg", hm1.createHeatmap());
 
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 7), new ConfigHeatmap());
+        Heatmap hm2 = new Heatmap(new TrialData(ToolType.EYETRACKING, "7", "3"), new ConfigHeatmap());
         assertEquals("createHeatMap sollte .../heatmap/HEATMAP_EYETRACKING_7.jpg ausgeben",
                 ".../heatmap/HEATMAP_EYETRACKING_7.jpg", hm2.createHeatmap());
 
-        Heatmap hm3 = new Heatmap(new Data(ToolType.CODECHARTS, 3), new ConfigHeatmap());
+        Heatmap hm3 = new Heatmap(new TrialData(ToolType.CODECHARTS, "3", "3"), new ConfigHeatmap());
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 101).